ddtrace/tracer.py

```diff
@@ -581,6 +580,9 @@ def _start_span(
                 # We must always mark the span as sampled so it is forwarded to the agent
                 span.sampled = True
 
+        if not span._parent:
+            span.meta["runtime-id"] = get_runtime_id()
```

Comment:
I wonder if this should go into `TraceTagsProcessor` instead?

Reply:
unless, does profiling need to access this at all? I know we need to use the same `runtime-id`, but profiling doesn't read it from the span does it?

Reply:
profiling uses the `runtime` API to get the runtime id. 

> I wonder if this should go into TraceTagsProcessor instead?

Hmm yeah this is doable. A SpanProcessor would probably make more sense. I think most of the tags set in `start_span` could be moved out to a `DefaultTagsSpanProcessor` or similar.

I'm ok with merging this as-is and making the refactor after WDYT @brettlangdon?

Reply:
well, the question is whether it should be put on trace root, chunk root, or all spans?

I think chunk root makes the most sense right now, at least until we have a more optimized way to associate a tag with all spans.

but I am ok with taking this as-is, and then follow-up moves to chunk root or all spans.

Reply:
I would think that it's like service where all spans under a span with a defined runtime id will share the runtime id. We should check with the backend teams to see what it looks for.

Reply:
yeah, we can check.